pkg/playback/http_playback_test.go

```diff
@@ -140,19 +138,18 @@ func TestSimpleRecordReplayServerSeparately(t *testing.T) {
 	assert.Equal(t, mockResponse2.StatusCode, res2.StatusCode)
 
 	// Shutdown record server
-	resShutdown, err := http.Post("http://localhost:8080/playback/stop", "text/plain", nil)
+	resShutdown, err := http.Get("http://localhost:8080/playback/cassette/eject")
 	server.Shutdown(context.TODO())
 	assert.NoError(t, err)
 	defer resShutdown.Body.Close()
 
 	// --- Set up a replay server
-	httpReplayer := newReplayServer(webhookURL)
-	err = httpReplayer.readCassette(&cassetteBuffer)
+	httpWrapper, err = NewServer(remoteURL, webhookURL, "/tmp", Replay, "cassette.yaml")
```

Comment:
might want to change this dir name to use `os.TempDir` to ensure Windows tests pass

Reply:
oh good point!